### ISSUE TITLE
Crash on "functionapp list"

### ISSUE THREAD

**Issue by cyplo** [Azure/azure-functions-cli#25]
```
PS C:\Users\cyryl> func azure functionapp list
System.NullReferenceException: Object reference not set to an instance of an object.
   at ARMClient.Library.ArmUriTemplate.<>c__DisplayClass6_0.<Bind>b__2(String a, String b)
   at System.Linq.Enumerable.Aggregate[TSource,TAccumulate](IEnumerable`1 source, TAccumulate seed, Func`3 func)
   at ARMClient.Library.ArmUriTemplate.Bind(Object obj)
   at WebJobs.Script.Cli.Arm.ArmManager.<GetFunctionAppsAsync>d__35.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   at WebJobs.Script.Cli.Actions.AzureActions.ListFunctionAppsAction.<RunAsync>d__3.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   at WebJobs.Script.Cli.ConsoleApp.<RunAsync>d__4`1.MoveNext()
```

**Comment by ahmelsayed** [comment]
this should be fixed with the fix to https://github.com/Azure/azure-functions-cli/issues/14 though it hasn't been released yet. I'll get it out soon for you to try.

**Comment by cyplo** [comment]
Thank you ! Looking forward to a new version !

**Comment by lindydonna** [comment]
Closed with the fix to #14
